Application/EdFi.Ods.AdminApi.AdminConsole/Helpers/Encryption.cs

```diff
@@ -0,0 +1,35 @@
+// SPDX-License-Identifier: Apache-2.0
+// Licensed to the Ed-Fi Alliance under one or more agreements.
+// The Ed-Fi Alliance licenses this file to you under the Apache License, Version 2.0.
+// See the LICENSE and NOTICES files in the project root for more information.
+
+using Rijndael256;
```

Comment:
We need to change this. Thankfully we now have the encryption code that was brought in for the connection strings. Please create a ticket for removing Rijndael256 and replacing with only the AES256 method used for connection strings.

"Rijndael" is another name for the AES algorithm, but the `Rijndael` NuGet package is very old and we can rely on the .NET framework now.

Reply:
[ADMINAPI-1162](https://edfi.atlassian.net/browse/ADMINAPI-1162)